From 2cb2f8a419f10c5cf808572c80f51c1df09d1de0 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 23 Jun 2016 16:10:29 -0400
Subject: [PATCH 52/55] Fix a typo on the rhel-7.3 branch...

Apparently introduced an error resolving a conflict on this branch,
woops.

Related: rhbz#1285601
---
 grubby.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grubby.c b/grubby.c
index 0c260f2..2402116 100644
--- a/grubby.c
+++ b/grubby.c
@@ -4286,7 +4286,7 @@ int addNewKernel(struct grubConfig *config, struct singleEntry *template,
 		 const char *newKernelPath, const char *newKernelTitle,
 		 const char *newKernelArgs, const char *newKernelInitrd,
 		 const char **extraInitrds, int extraInitrdCount,
-		 const char *newMBKernel, const char *newMBKernelArgs
+		 const char *newMBKernel, const char *newMBKernelArgs,
 		 int newIndex)
 {
 	struct singleEntry *new, *entry, *prev = NULL;
-- 
2.5.5

